docs/swagger.yaml

```diff
@@ -0,0 +1,329 @@
+swagger: "2.0"
+info:
+  description: "API Documentation for Users/Tasks"
+  version: "1.0.0"
+  title: "Users/Task API"
+  termsOfService: "http://swagger.io/terms/"
+  contact:
+    email: "apiteam@swagger.io"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "nsc-fun-dev-usw2-thursday.azurewebsites.net"
+
+schemes:
+- "https"
+- "http"
+paths:
+  /users:
+    get:
+      tags:
+      - "users"
+      summary: "Get list of all users"
+      description: ""
+      operationId: "getUsers"
+      produces:
+      - "application/xml"
+      - "application/json"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/User"
```

Comment:
Isn't there a specific way to denote that this will be an array of Users objects?